lte/gateway/python/integ_tests/s1aptests/test_3485_timer_for_default_bearer_with_mme_restart.py

```diff
@@ -53,7 +53,6 @@ def test_3485_timer_for_default_bearer_with_mme_restart(self):
         self._s1ap_wrapper.configUEDevice(num_ue)
         req = self._s1ap_wrapper.ue_req
         ue_id = req.ue_id
-
```

Comment:
Just out of curiosity, was this change done by the formatter?

Reply:
no just added some debug code at this points and accidential removed these lines.